src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -398,52 +429,41 @@ public void AskExit()
         exitConfirmationDialog.PopupCenteredShrink();
     }
 
-    private void InitConfigControls()
+    private void InitNewWorld(IAutoEvoConfiguration configuration)
     {
-        allowSpeciesToNotMutateCheckBox.Pressed = autoEvoConfiguration.AllowSpeciesToNotMutate;
-        allowSpeciesToNotMigrateCheckBox.Pressed = autoEvoConfiguration.AllowSpeciesToNotMigrate;
-        biodiversityAttemptFillChanceSpinBox.Value = autoEvoConfiguration.BiodiversityAttemptFillChance;
-        biodiversityFromNeighbourPatchChanceSpinBox.Value = autoEvoConfiguration.BiodiversityFromNeighbourPatchChance;
-        biodiversitySplitIsMutatedCheckBox.Pressed = autoEvoConfiguration.BiodiversityNearbyPatchIsFreePopulation;
-        biodiversityNearbyPatchIsFreePopulationCheckBox.Pressed = autoEvoConfiguration.BiodiversitySplitIsMutated;
-        lowBiodiversityLimitSpinBox.Value = autoEvoConfiguration.LowBiodiversityLimit;
-        maximumSpeciesInPatchSpinBox.Value = autoEvoConfiguration.MaximumSpeciesInPatch;
-        moveAttemptsPerSpeciesSpinBox.Value = autoEvoConfiguration.MoveAttemptsPerSpecies;
-        mutationsPerSpeciesSpinBox.Value = autoEvoConfiguration.MutationsPerSpecies;
+        worldsList.Add(new AutoEvoExploringToolWorld(configuration));
+        WorldsListMenuIndexChanged(worldsList.Count - 1);
+
+        worldsListMenu.AddItem((worldsList.Count - 1).ToString(), false, Colors.White);
```

Comment:
CurrentCulture should probably be used in the ToString call here? I assume that's meant to be user readable.